packages/components/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @hashicorp/design-system-components
 
+## 0.13.0
+
+### Minor Changes
```

Comment:
@alex-ju do you think this should be a `minor` or a `patch`? (considered that our current convention, until v. 1.0.0 is releases, is that `minor` is like a `major`)? I don't think we changed the API of the component, we just expanded it.

@Dhaulagiri what do you think?

Reply:
If I'm being pedantic, the dropdown component didn't have the `close` functionality exposed before this change, hence we introduce a new feature in this component. However, if we think of this feature as something that was originally intended to be part of the component, but missed in implementation, we can see it as a fix/patch.
Happy to either of the approaches so I'll defer the decision to @Dhaulagiri.

Reply:
I could go either way 🤷  Technically it's adding new api vs "patching" something that already existed. Ultimately I don't think it matters _too_ much as consumers will need to update either way.

Reply:
Having a look at previous releases made me stick with a patch, for consistency.